tests/wdio/resize-spec.js

```diff
@@ -0,0 +1,21 @@
+/* global browser, describe, it, expect, before, Terra */
+describe('resize utility', () => {
+  before(() => browser.url('/compare.html'));
+
+  it('resizes to tiny and huge', () => {
+    Terra.resize('tiny', 'huge', (name, viewport) => {
```

Comment:
Yesterday that webdriver.io commands have to exist within an it() or before/after() hook so we can't wrap a bunch of it() calls in a resize helper. And then looking at this test and the resize implementation, I am wondering if we are providing much value by providing this helper. Webdriver's `setViewportSize()` is simple to implement and `should.beAccessible()` and `should.beComparable()` accept an array of viewport sizes.

Reply:
yeah, i'm fine if we want to get rid of this helper. It was a worthy experiment to try it out

Reply:
If we could do multiple nesting of its() I think it could be valuable. Good to know this concept works if needed later.